msteams-platform/includes/tabs/content-security-policy-headers.md

```diff
@@ -3,10 +3,10 @@
 >
 > 1. Update TeamsJS SDK to v.2.19.0 or higher. For more information about the latest release of TeamsJS SDK, see [Microsoft Teams JavaScript client library](https://www.npmjs.com/package/@microsoft/teams-js).
 >
-> 2. Update your [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) headers in your Teams app to allow your app to access the ***teams.cloud.microsoft*** domain. If your Teams app extends across Outlook and Microsoft 365, ensure you allow your app to access ***teams.cloud.microsoft***, ***outlook.cloud.microsoft***, and ***m365.cloud.microsoft*** domains.
+> 2. Update your [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) headers in your Teams app to allow your app to access the ****.cloud.microsoft*** domain. This will ensure your app continues to work across existing and future Microsoft 365 host applications, and minimize changes required of your application in the future.
```

Comment:
@erikadoyle, can we modify the sentence to avoid using past and future tense in line with Microsoft Style Guide?

**Suggestion**: This ensures your app continues to work across existing and future Microsoft 365 host applications and minimizes the need for subsequent changes.

Reply:
Good suggestion! Incorporated.